Dockerfile

```diff
@@ -14,6 +12,12 @@ USER zulip
 
 RUN ln -nsf /srv/zulip ~/zulip
 
-RUN echo 'export LC_ALL="en_US.UTF-8"' >> ~zulip/.bashrc
+RUN echo 'export LC_ALL="en_US.UTF-8" LANG="en_US.UTF-8" LANGUAGE="en_US.UTF-8"' >> ~zulip/.bashrc
 
 WORKDIR /srv/zulip
+
+CMD ["/usr/bin/python /srv/zulip/tools/provision.py","--docker"]
+CMD ["source /srv/zulip-venv/bin/activate"]
+CMD ["./srv/zulip/tools/start-dockers"]
```

Comment:
That, or no `.` at the start.  @stanzheng does this PR somehow not match what you tested?